examples/slash_commands/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"flag"
+	dgo "github.com/bwmarrin/discordgo"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+var (
+	botToken  = flag.String("bot_tok", "", "Bot token")
```

Comment:
`t` and `g` instead of `bot_tok` and `test_guild` to be consistent with flag names of other examples.